the gettext tooling does not quite understand 4 version components.
hack it so the output in the configure script sticks to 3 parts.
this is a horrible hack that really needs an upstream fix.

https://lists.gnu.org/archive/html/bug-gettext/2015-07/msg00027.html
https://bugs.gentoo.org/556024

--- a/gettext-tools/misc/gettextize.in
+++ b/gettext-tools/misc/gettextize.in
@@ -1235,7 +1235,7 @@
 func_modify_configure_in "(AC_OUTPUT): Remove command that created po/Makefile."
 sed -e '/^\(dnl \|\)AC_LINK_FILES(\$nls_cv_header_libgt, \$nls_cv_header_intl)$/d' < "$srcdir/$configure_in" > "$srcdir/$configure_in.tmp"
 func_modify_configure_in "(AC_LINK_FILES): Remove invocation."
-sed -e 's/^AM_GNU_GETTEXT_VERSION([^()]*)/AM_GNU_GETTEXT_VERSION(['"$version"'])/' < "$srcdir/$configure_in" > "$srcdir/$configure_in.tmp"
+sed -e 's/^AM_GNU_GETTEXT_VERSION([^()]*)/AM_GNU_GETTEXT_VERSION(['"${version%.?}"'])/' < "$srcdir/$configure_in" > "$srcdir/$configure_in.tmp"
 func_modify_configure_in "(AM_GNU_GETTEXT_VERSION): Bump to $version."
 $do_changelog && func_ChangeLog_finish
 
